index.html

```diff
@@ -1930,6 +1930,10 @@ <h2 class="title-section">Proudly sponsored by</h2>
 
                <div class="sponsors-list-wrapper">
                    <div class="sponsors-list" id="sponsors-carousel" style="text-align: center">
+                   		<!-- item -->
+                       <div class="sponsor-item platinum">
+                        <a href=""><img src="img/microsoft-logo.jpg" alt="" class="sponsor-img"> </a>
```

Comment:
Ideally it should've contained the url as well as alt text.

@rajataaron @akshayaurora